### PR TITLE
fixed mnist url in kubernets [EKS/AKS/GKE] documentation

### LINKED ISSUE #895: Deploy TorchServe using Helm Charts fails for MNIST example in EKS
## Context
Following the EKS doc, deploying torchserve with hellm charts, checking the ts logs mnist example failed with the error: 'MNISTDigitClassifier' object has no attribute 'handle'. Removing mnist from config.properties, squeezenet worked fine, replacing mnist with densenet also worked fine. Logs are attached. 

* torchserve version: 0.3.0b20201214
* torch-model-archiver version: 0.2.1b20201214
* torch version:1.7
* torchvision version [if any]:
* torchtext version [if any]:
* torchaudio version [if any]:
* java version:
* Operating System and version:ubuntu 18.04

## Your Environment

* Installed using source? [yes/no]:
* Are you planning to deploy it using docker container? [yes/no]:
* Is it a CPU or GPU environment?: GPU
* Using a default/custom handler? [If possible upload/share custom handler/model]:
* What kind of model is it e.g. vision, text, audio?: vision 
* Are you planning to use local models from model-store or public url being used e.g. from S3 bucket etc.?
  [If public url then provide link.]:
* Provide config.properties, logs [ts.log] and parameters used for model registration/update APIs:
* Link to your project [if any]:

## Expected Behavior
register the model

## Current Behavior
model registration fails with error:
'MNISTDigitClassifier' object has no attribute 'handle'

## Steps to Reproduce

Following EKS doc on ubuntu 18.04, EC2 gpu instance (cluster , p2.xlarge)

## Failure Logs [if any]

Logs include the error of mnist, removing the mnist and running squeezenet fine. Logs of replacing the mnist with densent.
[EKS_logs_gpu_mnist_failed_squeezenet_worked.logs.txt](https://github.com/pytorch/serve/files/5686859/EKS_logs_gpu_mnist_failed_squeezenet_worked.logs.txt)
[Rpelacing_mnist_with_densent_working.logs.txt](https://github.com/pytorch/serve/files/5686860/Rpelacing_mnist_with_densent_working.logs.txt)

### PR DESCRIPTION
**Pull request by harshbafna**
## Description

Fixes #895 

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Feature/Issue validation/testing

Please describe the tests [UT/IT] that you ran to verify your changes and relevent result summary. Provide instructions so it can be reproduced. 
Please also list any relevant details for your test configuration.

- [ ] Test A
- [ ] Test B

- UT/IT execution results

- Logs

## Checklist:

- [ ] Have you added tests that prove your fix is effective or that this feature works?
- [ ] New and existing unit tests pass locally with these changes?
- [ ] Has code been commented, particularly in hard-to-understand areas?
- [ ] Have you made corresponding changes to the documentation?

### CODE REVIEW

**Review by HamidShojanazeri — COMMENTED** [review]
@harshbafna Thanks, it looks fine to me, just we need to add that image is required to be built from src or changing the referenced image as discussed in  [issue #895](https://github.com/pytorch/serve/issues/895#issuecomment-744278414)

**Review by chauhang — APPROVED** [review]

File: kubernetes/AKS/README.md [hunk 1]
```diff
@@ -118,11 +118,11 @@ nvidia-device-plugin-daemonset-qccgn   1/1     Running   0          144m
 
 ```bash
 wget https://torchserve.pytorch.org/mar_files/squeezenet1_1.mar
-wget https://torchserve.pytorch.org/mar_files/mnist.mar
+wget https://torchserve.pytorch.org/mar_files/mnist_v2.mar
```
Comment (maaquib) [inline kubernetes/AKS/README.md:121]: @harshbafna The https://torchserve.pytorch.org/mar_files/mnist.mar is currently pointing to the new mar file generated by @abishekchiffon 
`mnist_v2` doesn't contain any envelope changes.

Comment (harshbafna) [reply kubernetes/AKS/README.md:121]: @maaquib: Thanks for the update.

@HamidShojanazeri: Could you validate the https://torchserve.pytorch.org/mar_files/mnist.mar with updated docker image built from source?

If that works fine I will revert these changes and make the changes in the model zoo with the new URL.

**Comment by harshbafna** [comment]
@HamidShojanazeri: I checked, both mnist.mar and mnist_v2.mar are exactly the same now and the root cause of #895 is the outdated docker image, which should get fixed with 0.3.0 release.

There are no changes required in the model zoo either.

Closing this PR.

**Pull request closed without merging**